### PR TITLE
RPM & DEB packaging

### PR DESCRIPTION
**Pull request by bevhost**

### CODE REVIEW

File: pdns-dynamic-reverse-backend.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 Copyright (c) 2010 Stefan "ZaphodB" Schmidt
 Copyright (c) 2011 Endre Szabo
 Copyright (c) 2017 Technical University of Munich (Lukas Erlacher)
+Copyrught (c) 2019 David Beveridge (bevhost)
```
Comment (endreszabo) [inline pdns-dynamic-reverse-backend.py]: You have a typo here.

File: pdns-dynamic-reverse-backend.py [hunk 2]
```diff
@@ -48,10 +54,10 @@
 import yaml
 
 LOGLEVEL = 2
-CONFIG = 'dynrev.yml'
+CONFIG = '/etc/pdns/dynrev.yml'
```
Comment (endreszabo) [inline pdns-dynamic-reverse-backend.py]: You passed this file path as argument in config `pipe-handler` option. Use that (`argv[1]`) instead IMO.

File: pdns-dynamic-reverse-backend.py [hunk 3]
```diff
@@ -7,9 +7,14 @@
 pdns.conf example:
 
 launch=pipe
-pipe-command=/usr/local/sbin/pipe-local-ipv6-wrapper
+pipe-command=/usr/sbin/pdns-dynamic-reverse-backend.py /etc/pdns/dynrev.yml
 pipe-timeout=500
 
+
```
Comment (endreszabo) [inline pdns-dynamic-reverse-backend.py]: This is an extra newline.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+clean: 
+
+
+install: 
+	$(MKDIR_P) $(DESTDIR)$(ETCDIR)
+	$(MKDIR_P) $(DESTDIR)$(BINDIR)
+	$(MKDIR_P) $(DESTDIR)$(DOCDIR)
+	$(INSTALL) -pm 644 dynrev.yml $(DESTDIR)$(ETCDIR)/dynrev.yml.example
+	$(INSTALL) -pm 644 README.md $(DESTDIR)$(DOCDIR)/README
+	$(INSTALL) -pm 750 pdns-dynamic-reverse-backend.py $(DESTDIR)$(BINDIR)/pdns-dynamic-reverse-backend.py
```
Comment (endreszabo) [inline Makefile]: You defined perms as `750`. User running `pdns_server` will not be able to access it.

**Review by endreszabo — CHANGES_REQUESTED** [review]
Thanks for the effort you put in :)

Pls check my comments.

**Comment by bevhost** [comment]
Please check out my latest commits

**Comment by endreszabo** [comment]
Well done, thank you for all the efforts you put in this PR.

**Pull request merged**